src/docs/composing_applications.md

```diff
@@ -1,11 +1,16 @@
 ---
-title: Build your own IDE
+title: Build your own IDE/Tool
 ---
 
 
-# Build your own IDE
+# Build your own IDE/Tool
 
-This guide will teach you how to build your own Theia application.
+This guide will teach you how to build your own Theia-based application. In a nutshell, you will configure which existing components will initially be part of your product. These components can be Theia extensions or VS Code extensions. Please get familiar with the [extension mechanisms of Theia](https://theia-ide.org/docs/extensions/) in case you are not already.
```

Comment:
I'm not sure about the sentence 

> In a nutshell, you will configure which existing components will initially be part of your product.

I don't think we should say existing or initially here, what exactly are we trying to say?

I think we can elaborate to be clearer:

```suggestion
This guide will teach you how to build your own Theia-based application.The guide will demonstrate how to configure your own application composed of existing or new Theia extensions, and any VS Code extensions you want bundled in your application by default. Please get familiar with the [extension mechanisms of Theia](https://theia-ide.org/docs/extensions/) in case you are not already.
```

Reply:
great!